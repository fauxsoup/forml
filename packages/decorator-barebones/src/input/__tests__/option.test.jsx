import Option from '../option';
import React from 'react';
import { render } from '@testing-library/react';

describe('renders', function () {
    test('its children', function () {
        const { container } = render(<Option value={1}>1</Option>);

        expect(container).toMatchSnapshot();
        expect(container.querySelector('option')).not.toBeNull();
        expect(container.querySelector('option').textContent).toBe('1');
    });
});
