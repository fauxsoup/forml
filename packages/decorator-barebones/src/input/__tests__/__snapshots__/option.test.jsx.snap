// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders its children 1`] = `
<div>
  <option
    value="1"
  >
    1
  </option>
</div>
`;
